.travis/before_install.sh

```diff
@@ -96,14 +99,6 @@ if [ -z "$TRAVIS_TAG" ]; then
 fi
 
 
-git clone --depth=1 https://github.com/pulp/pulp_ansible.git --branch master
-if [ -n "$PULP_ANSIBLE_PR_NUMBER" ]; then
-  cd pulp_ansible
-  git fetch --depth=1 origin pull/$PULP_ANSIBLE_PR_NUMBER/head:$PULP_ANSIBLE_PR_NUMBER
-  git checkout $PULP_ANSIBLE_PR_NUMBER
-  cd ..
-fi
-
```

Comment:
Removing `pulp_ansible` from `additional_plugins` will remove this piece of code